Source/TolgeeEditor/Private/TolgeeEditorIntegrationSubsystem.cpp

```diff
@@ -20,6 +20,12 @@
 #include <Serialization/JsonInternationalizationManifestSerializer.h>
 #include <Settings/ProjectPackagingSettings.h>
 
+#include "ISourceControlModule.h"
+#include "ISourceControlProvider.h"
+#include "ISourceControlOperation.h"
+#include "ISourceControlState.h"
+#include "SourceControlOperations.h"
```

Comment:
```suggestion
#include <Settings/ProjectPackagingSettings.h>
#include <ISourceControlModule.h>
#include <ISourceControlProvider.h>
#include <ISourceControlOperation.h>
#include "ISourceControlState.h"
#include "SourceControlOperations.h"
```

Sorry, I know it's not conventional, but to keep things separated and consistent, we are using `<>` for engine includes and `""` for plugin includes.